src/plugins/advanced_settings/public/management_app/advanced_settings.tsx

```diff
@@ -197,6 +201,8 @@ export class AdvancedSettingsComponent extends Component<
           value: setting[1].userValue,
           isCustom: config.isCustom(setting[0]),
           isOverridden: config.isOverridden(setting[0]),
+          isPermissionControlled:
+            all[setting[0]].scope === UiSettingScope.DASHBOARD_ADMIN && !isDashboardAdmin,
```

Comment:
What about the case when permission feature flag is turned off?

Reply:
Refer to https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/server/plugin.ts#L288  when the flag is off, every user will be dashboard admin, should I add a comment here?

Reply:
Oh, cool then. My bad memory.